src/index.html

```diff
@@ -11,8 +11,61 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <main
+      class="card"
+      data-qa="card"
+    >
+      <img
+        src="./images/imac.jpeg"
+        class="card__photo"
+        alt="imac"
+      />
+      <div class="card__description">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
+      <div class="card__code">Product code: 195434</div>
+      <div class="stars stars--4">
+        <section class="stars__blok">
+          <article class="stars__active"></article>
+          <article class="stars__active"></article>
+          <article class="stars__active"></article>
+          <article class="stars__active"></article>
+          <article class="stars__not_active"></article>
+        </section>
+        <div class="stars__review">Reviews: 5</div>
+      </div>
+      <div class="card__price">
+        <div class="card__price_1">Price:</div>
+        <div class="card__price_2">$2,199</div>
+      </div>
+      <div class="card__buy">
+        <a
+          href="http://"
```

Comment:
The href attribute for the 'Buy' link is set to 'http://', which is not a valid URL. Consider updating it to a valid URL or use a placeholder like '#' if the URL is not yet determined.